cves/CVE-2015-8662.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function that decodes JPEG2000 compression does not check the level of 
+  compression on the JPEG2000 file being decompressed. This can lead to a situation
+  where a specially crafted JPEG2000 file can be manipulated to cause an index out
+  of bounds error. This index out of bounds can be used by an attacker who has
```

Comment:
How would you be able to manipulate the JPEG2000 file in order to get this error?